web/satellite/src/components/accessGrants/permissions/BucketsSelection.vue

```diff
@@ -33,6 +33,10 @@ import ExpandIcon from '@/../static/images/common/BlackArrowExpand.svg';
     },
 })
 export default class BucketsSelection extends Vue {
+    @Prop({default: ''})
+    private readonly containerStyle: string;
```

Comment:
you can just apply styles for component in parent. no need to pass style props
for exceptions u can use ::v-deep

Reply:
Completed:

    ::v-deep .buckets-selection {
        margin-left: 0;
        height: 30px;
        border: 1px solid #c8d3de;
    }

    ::v-deep .buckets-selection__toggle-container {
        padding: 10px 20px;
    }